### PR TITLE
T9570 add equal and not equal to field in satpam

### PR DESCRIPTION
**Pull request by nedbertus**

### CODE REVIEW

File: src/locale/id.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ module.exports = {
   'email': 'Format email tidak valid.',
   'emptyString': 'Input harus kosong',
   'equal:$1': 'Input tidak sesuai.',
+  'equal-to-field:$1': 'Input tidak sesuai.',
```
Comment (sendyhalim) [inline src/locale/id.js]: Shouldn't this `Input harus sama dengan <%= inputObj[ruleParams[0]] %>` ?

**Review by sendyhalim — CHANGES_REQUESTED** [review]

**Comment by sendyhalim** [comment]
Hi, thanks for the PR. There's some locale wordings that I think can be improved plus you need to bump the version to `2.8.0` https://semver.org/

File: src/locale/id.js (hunk 1 above)
Comment (nedbertus) [reply src/locale/id.js]: done

**Comment by nedbertus** [comment]
> Hi, thanks for the PR. There's some locale wordings that I think can be improved plus you need to bump the version to `2.8.0` https://semver.org/

done

**Comment by sendyhalim** [comment]
Thanks!

**Pull request merged**